### PR TITLE
MNT Remove TODO comments

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/.github/issues/116

### CODE REVIEW

File: client/src/containers/CampaignAdmin/CampaignAdmin.js [hunk 1]
```diff
@@ -92,7 +92,6 @@ class CampaignAdmin extends Component {
         // NOOP - Lazy loaded in CampaignAdminList.js
         break;
       case 'edit':
-        // @todo - Lazy load in FormBuilderLoader / GridField
```
Comment (sabina-talipova) [inline client/src/containers/CampaignAdmin/CampaignAdmin.js:95]: ~~https://github.com/silverstripe/silverstripe-campaign-admin/issues/281~~

File: src/CampaignAdmin.php
Comment (sabina-talipova) [inline src/CampaignAdmin.php:1]: - https://github.com/silverstripe/silverstripe-campaign-admin/issues/282

File: client/src/containers/CampaignAdmin/CampaignAdmin.js (hunk 1 above)
Comment (GuySartorelli) [reply client/src/containers/CampaignAdmin/CampaignAdmin.js:95]: That's not the right issue

Comment (sabina-talipova) [reply client/src/containers/CampaignAdmin/CampaignAdmin.js:95]: Sorry, there is no ticket for this TODO

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**